components/omega/doc/devGuide/HorzMesh.md

```diff
@@ -0,0 +1,53 @@
+(omega-dev-horz-mesh)=
+
+## Horizontal Mesh
+
+The OMEGA horizontal mesh uses the [MPAS Mesh
+Specification](https://mpas-dev.github.io/files/documents/MPAS-MeshSpec.pdf).
+A mesh object is created by the `init` method, which assumes that `Decomp` has
+already been initialized.
+```c++
+Err = OMEGA::Decomp::init();
+Err = OMEGA::HorzMesh::init();
+```
+The constructor replicates the subdomain mesh cell/edge/vertex counts and
+connectivity information from Decomp so this information can be passed among the
+computational routines, alongside the other local mesh information.  It then
+creates several parallel I/O decompositions and reads in the remaining subdomain
+mesh information.  Finally, any mesh information needed on the device to a
+device YAKL array from the host.  These tasks are organized into several private
```

Comment:
There is something missing in the sentence beginning with "Finally" ("is copied to" ?). It might be a good idea to explicitly mention in this guide that the coordinate arrays are not available on the device.

Reply:
Thanks @mwarusz!